src/web/components/PatientView/PrintRecord/PrintPathologyForm.js

```diff
@@ -117,13 +147,31 @@ export default ({
       </div>
 
       <div className="columns">
-        <div className="column">
-          <Block data={record} block={style[0]} />
-        </div>
-        <div className="column">
-          <Block data={record} block={style[1]} />
-        </div>
+      {
+        (style.slice(0, 2)).map((element, index) => (
+          <div className="column" key={index}>
+            <Block data={record} block={element} />
+          </div>
+        ))}
       </div>
+      {(() => {
+        let testtype;
```

Comment:
(There is no problem. This is just an advice or suggestion)

It seems a variable `testtype` can be moved to inside the for loop like this:

```
...
for (let i = 2; i < style.length; i += step) {
  const testtype = (...);
}
```

This makes the scope of the variable narrow.

Reply:
Indeed! Thanks, I missed that :) was planning to make it used after the loop initially and forgot to narrow the scope.